xlators/mgmt/glusterd/src/glusterd-bitrot.c

```diff
@@ -88,7 +88,7 @@ __glusterd_handle_bitrot(rpcsvc_request_t *req)
     ret = dict_get_str(dict, "volname", &volname);
     if (ret) {
         snprintf(msg, sizeof(msg), "Unable to get volume name");
-        gf_msg(this->name, GF_LOG_ERROR, errno, GD_MSG_DICT_GET_FAILED,
+        gf_msg(this->name, GF_LOG_ERROR, ret, GD_MSG_DICT_GET_FAILED,
```

Comment:
nit: we have >500 GD_MSG_DICT_SET_FAILED messages. Would be great to just (in a different patch!) turn them into a macro that uses gf_smsg() and the key name.

Reply:
sure